added latest improvement for the load testing (#1323)
saucelabs/sauce-docs: docs/api-testing/load-testing.md

* Start - The start date and time of the test.
* End - The end date and time of the test.
* User - The user that executed the test.
* Environment - The environment selected for running the test (if any)
* Agents - The agents used to run the test.
* Virtual Users - The number of virtual users for the execution of the test.

The report contains a dropdown that shows all the endpoints the test is calling that you can use to filter to a specific one (the default value shows the data for All the endpoints). For best results, set up footprints to ensure the report is readable. For more information, see [Improving Metrics](/api-testing/composer#improving-metrics).

<img src={useBaseUrl('/img/api-testing/load-test-footprint.png')} alt="The Footprint dropdown" width="200"/>

> The Summary dynamically shows totals of Response Time (Average, 90th percentile, maximum and minimum), Requests (Total requests and total failures), and Status Codes based on the selection in the footprint dropdown.

Reviewer: ```suggestion
The Summary dynamically shows totals of Response time (Average, 90th percentile, Maximum, and Minimum), Requests (Total requests and Failures), and Status codes based on the selected Footprint.
```

<img src={useBaseUrl('/img/api-testing/load-test-report-status-codes.png')} alt="Status codes graph" width="400"/>

> Once the test is completed, at the bottom of the report, "Failures" section will be shown. In this section, you will see all the most recent failures that happened during the test. The section shows one row for each test execution. Each row has the date and time, the number of failures, and a link to the specific report for that test. 

Reviewer: @LoryApiFortress Does the Failures section still show even if there are no test failures?

Author: @lysannep no, the section is shown only when test is completed AND there are failures, otherwise it is not shown

Reviewer: Ok, thanks for clarifying.

> Once the test is completed, at the bottom of the report, "Failures" section will be shown. In this section, you will see all the most recent failures that happened during the test. The section shows one row for each test execution. Each row has the date and time, the number of failures, and a link to the specific report for that test. 

Reviewer: ```suggestion
Once the test is completed, the Failures section will be shown at the bottom of the report. In this section, you will see all the failures that happened during the test. The section shows one row for each test execution. Each row has the date and time, the number of failures, and a link to the specific report for that test. 
```

> The **Load more failures** allows you to load more failures. 

Reviewer: ```suggestion
```
This last sentence is a bit redundant. I suggest we remove it.

Author: @lysannep by default we show only 20 failures, the "load more failures" button allows user to load more results (in groups of 20)
